File: banda/AMarotoChivite-banda/AMarotoChivite-banda-Java/src/main/java/simulacionBandaMusica/factories/TeclistaFactory.java
package simulacionBandaMusica.factories;

import simulacionBandaMusica.models.Teclista;

import static simulacionBandaMusica.utils.sorteoDatos.*;

public class TeclistaFactory {

    public static Teclista create() {
        String name = randomName();
        int experienceYear = randomAnno();
        int numTeclas = randomNumTeclas();
        return new Teclista(name, "Teclado", experienceYear, numTeclas);
    }

}

File: banda/AMarotoChivite-banda/AMarotoChivite-banda-Kotlin/src/main/kotlin/simulacionBandaMusica/factories/TeclistaFactory.kt
package simulacionBandaMusica.factories

import simulacionBandaMusica.models.Musico
import simulacionBandaMusica.utils.sorteoDatos

object TeclistaFactory {
    fun create(): Musico.Teclista {
        val name = sorteoDatos.randomName()
        val experienceYear = sorteoDatos.randomAnno()
        val numTeclas = sorteoDatos.randomNumTeclas()
        return Musico.Teclista(name, "Teclado", experienceYear, numTeclas)
    }
}